bin/ci

```diff
@@ -69,14 +69,15 @@ prepare_system() {
   on_linux 'echo '"'"'{"ipv6":true, "fixed-cidr-v6":"2001:db8:1::/64"}'"'"' | sudo tee /etc/docker/daemon.json'
   on_linux sudo service docker restart
 
-  # TODO: Delete Me
-  on_linux cat /etc/passwd
-  on_linux cat /etc/group
-
   on_osx brew update
 }
 
 build() {
+
+  # TODO: Delete Me
+  with_build_env cat /etc/passwd
+  with_build_env cat /etc/group
```

Comment:
Needs to be in single quotes, or it'll just execute `cat` with no arg, which hangs forever.

Reply:
Oops